### ISSUE TITLE
Asynchronous event notification

### ISSUE THREAD

**Issue by vidicunt** [PhilippMundhenk/PulseAudioJava#5]
Hello! A feature such as this one:
https://github.com/janakj/pulseaudio.js/wiki/Event-Notification
would be very welcome!

**Comment by PhilippMundhenk** [comment]
Indeed, this would be a nice feature. I wasn't even aware PulseAudio provides events. With ```pactl subscribe```, it should be posible to pull them in here also.

I will see what I can do, but fair warning: I haven't touched this in ~5 years and don't currently have a test setup at hand, so this might take a lot of time. Contributions are appreciated, however.
